kf_lib_data_ingest/app/cli.py

```diff
@@ -150,14 +150,10 @@ def ingest(
 ):
```

Comment:
Can you keep the blob you removed and also keep what you added? I know thats redundant but we need that specially formatted argument description (using the `\b`) in order for the CLI to include it in its help text for positional arguments.